governance-exercises/Executive Vote 02.md

```diff
@@ -0,0 +1,162 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with links showing its provenance. "Authorization" links to the official document(s) authorizing the proposed item's inclusion in the Executive Vote; this can be a governance poll, the Atlas and/or a Scope Alignment Artifact. "Reasoning" links to a MakerDAO Forum post providing the rationale and additional context for each item. Where needed, "Authorizing Parties" may also be listed; these may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Enhanced Dai Savings Rate (EDSR) Activation
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmcTRPLx)\
+*Reasoning*: Rune's [Forum post](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405)
+
+The [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope) was recently amended to add a temporary [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-) system aimed at boosting Dai demand. The EDSR is determined based on the [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) and DSR utilization rate. If this executive proposal passes, the following action will occur:
+
+-  The Dai Savings Rate (DSR) will be increased by 4.81% from 3.19% to **8%**. 
+
+8% is the EDSR's upper limit, as prescribed in the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#3-2-2-1-edsr-upper-limit). Over time, as DSR utilization increases, the effective DSR will be adjusted downward.
+
+### Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling Increase
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmSLj3HS)\
+*Reasoning*: Phoenix Labs' [Forum post](https://forum.makerdao.com/t/phoenix-labs-proposed-changes-for-spark/21422)
+
+The proposed action concerns increasing the Spark D3M [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) parameter to prepare for the implementation of the [EDSR](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-) and allow new users to onboard onto the lending market. The [Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and [Ceiling Increase Cooldown](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) parameters will remain the same at 20 million DAI and 8 hours, respectively. If this executive proposal passes, the following action will occur:
```

Comment:
Arguably you don't need to include information about parameters remaining unchanged, that should be the default assumption.

Reply:
You maybe don't need to use bullet points where there is only one parameter change as well. Reads better as just: 

If this executive proposal passes, the Spark Protocol D3M (DIRECT-SPARK-DAI) Maximum Debt Ceiling will be increased from 20 million DAI to **200 million DAI**.